File: LeetCode_1935/Main.py
class Solution(object):
    def helper(self, word, brokenLetters):
        isBroken = False
        for ch in word:
            if ch in brokenLetters:
                isBroken = True
        return isBroken == False
    def canBeTypedWords(self, text, brokenLetters):
        """
        :type text: str
        :type brokenLetters: str
        :rtype: int
        """
        word = ""
        answer = 0
        for i in range(len(text)):
            if (text[i] == ' ') :
                answer += self.helper(word, brokenLetters)
                word = ""
            else:
                word += text[i]
        answer += self.helper(word, brokenLetters)
        return answer
        
            
            

File: LeetCode_1935/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    bool helper(string& word, unordered_set<char>& broken) {
        for (char c : word) {
            if (broken.count(c)) return false;
        }
        return true;
    }

    int canBeTypedWords(string text, string brokenLetters) {
        unordered_set<char> broken(brokenLetters.begin(), brokenLetters.end());
        int answer = 0;
        string word = "";

        for (int i = 0; i < text.size(); i++) {
            if (text[i] == ' ') {
                if (!word.empty() && helper(word, broken)) {
                    answer++;
                }
                word = "";
            } else {
                word += text[i];
            }
        }

        if (!word.empty() && helper(word, broken)) {
            answer++;
        }

        return answer;
    }
};

int main() {
    cout << true + true;
    return 0;
}
